### DISCUSSION TITLE
AdwEntryRow

### DISCUSSION THREAD

**Discussion by NoneE1** [Relm4/Relm4#222] [General]
Currently AdwEntryRow is available on libadwaita crate feature v1_2 
https://world.pages.gitlab.gnome.org/Rust/libadwaita-rs/stable/latest/docs/libadwaita/struct.EntryRow.html 
is there way to get that in relm4? 0.5-beta

**Comment by AaronErhardt** [comment]
Feature flags are additive AFAIK. If you import `libadwaita = { git = ..., features = ["v1_2"] }` you'll enable the feature in any other dependency that uses libadwaita too.

**Reply by NoneE1** [reply]
thats the first thing i tried
#cargo.toml
libadwaita = {git = "https://gitlab.gnome.org/World/Rust/libadwaita-rs", features = ["v1_2"]}

#error
warning: `"pkg-config" "--libs" "--cflags" "libadwaita-1" "libadwaita-1 >= 1.2.alpha"` did not exit successfully: exit status: 1

error: failed to run custom build command for `libadwaita-sys v0.2.0 (https://gitlab.gnome.org/World/Rust/libadwaita-rs#01881b0c)`
  error: could not find system library 'libadwaita-1' required by the 'libadwaita-sys' crate

  --- stderr
  Package dependency requirement 'libadwaita-1 >= 1.2.alpha' could not be satisfied.
  Package 'libadwaita-1' has version '1.1.2', required version is '>= 1.2.alpha'

**Reply by AaronErhardt** [reply]
But that's just because you don't have the right version of libadwaita installed on your system. It has nothing to do with Relm4.

**Reply by NoneE1** [reply]
Oh sorry didnt notice that im beginner :smile: building libadwaita 1.2.alpha fixed the issue Thanks
